hasher-matcher-actioner/hmalib/metrics/__init__.py

```diff
@@ -22,6 +22,19 @@
 logger = logging.getLogger(__name__)
 
 
+class lambda_with_datafiles:
```

Comment:
Nit: Class names are always UpperCamelCase

https://www.python.org/dev/peps/pep-0008/

LambdaWithDatafiles

All of the implementations just have a constant. What about this?

class LambdaWithDatafiles
  PREFIX = ""

  @property
    def download_datafiles(self):
        return f"{self.PREFIX}.download_datafiles"
  
class PDQIndexerLambda(LambdaWithDatafiles):
   PREFIX = "lambdas.pdqhasher"

Alternatively, just use dataclass and prefix: str

or alternatively this:
```
@dataclass
class LambdaName:
  prefix: str
  
  @property
  ...

pdq_indexer_lambda = LambdaName("prefix1")
whatever_lambda = LambdaName("prefix2")
```

Reply:
@jeberl the code looks like it stiil has `lambda_with_datafiles` instead of `LambdaWithDatafiles`. I know this PR is already merged at this point. What's the right process to take from here?